docs/MODULES.md

```diff
@@ -16,19 +16,26 @@ List of Module Documentations
 11. [Contrast](#contrast-module)
 12. [Convolution](#convolutioon-module)
 13. [DecodeQr](#decodeQr-module)
-14. [Dynamic](#dynamic-module)
-15. [Edge-Detect](#edge-detect-module)
-16. [Gamma-Correction](#gamma-correction-module)
-17. [Gradient](#gradient-module)
-18. [Histogram](#histogram-module)
-19. [Import-image](#import-image-module)
-20. [Invert](#invert-module)
-21. [Ndvi](#ndvi-module)
-22. [Ndvi-Colormap](#ndvi-colormap-module)
-23. [Overlay](#overlay-module)
-24. [Resize](#resize-module)
-25. [Rotate](#rotate-module)
-26. [Saturation](#saturation-module)
+14. [Dither](#dither-module)
+15. [DrawRectangle](draw-rectangle-module)
+16. [Dynamic](#dynamic-module)
+17. [Edge-Detect](#edge-detect-module)
+18. [Gamma-Correction](#gamma-correction-module)
+19. [Gradient](#gradient-module)
+20. [Histogram](#histogram-module)
+21. [Import-image](#import-image-module)
+22. [Invert](#invert-module)
+23. [Ndvi](#ndvi-module)
+24. [Ndvi-Colormap](#ndvi-colormap-module)
+26. [Overlay](#overlay-module)
+27. [PaintBucket](paint-bucket-module)
+28. [Resize](#resize-module)
+29. [Rotate](#rotate-module)
+30. [Saturation](#saturation-module)
+31. [Threshold](#threshold-module)
```

Comment:
Uh Oh. Seems like the Threshold, White Balance, and Tint links are broken. The way these links work is that they link to the title in the markdown file

```
## Title
```
This is a title. It will be shown in big bold letters. If you want to link to it you have  to write
`[link-text](#title)`. The text which comes after the '#' in the link is the title. It should be in all small letters and any spaces in the title should be converted to hyphens `-`

e.g.
```
## White Balance
```
A link to that title will be `[WhiteBalance](#white-balance)`

Reply:
```suggestion
31. [Threshold](#threshold)
```

This should be the link for threshold as the title text is `Threshold`